stage1/init/common/seccomp_wildcards.go

```diff
@@ -392,10 +394,26 @@ var (
 		"vm86old",
 	}
 
+	//RktDefaultSeccompArmWhitelist contains the additional needed syscalls for arm support
+	RktDefaultSeccompArmWhitelist = []string{
+		"arm_fadvise64_64",
+		"arm_sync_file_range",
+		"breakpoint",
+		"cacheflush",
+		"set_tls",
+		"sync_file_range2",
+	}
+
 	// RktDefaultSeccompBlacklist contains a default blacklist of syscalls,
 	// used by rkt for seccomp filtering.
 	RktDefaultSeccompBlacklist = DockerDefaultSeccompBlacklist
 	// RktDefaultSeccompWhitelist contains a default whitelist of syscalls,
 	// used by rkt for seccomp filtering.
 	RktDefaultSeccompWhitelist = DockerDefaultSeccompWhitelist
 )
+
+func init() {
+	if runtime.GOARCH == "arm" {
```

Comment:
I see we are also doing 64bits whitelisting here, so this should probably also check for the `arm64` case.